restic/internal/cat.py

```diff
@@ -16,5 +16,9 @@ def config(self):
         cmd = self.base_command() + ['cat', 'config']
         return self.run(cmd)
 
+    def lock(self, lock_id):
+        cmd = self.base_command() + ['cat', 'lock', lock_id, '--no-lock']  # pylint: disable=C0301
```

Comment:
Why do we need the pylint disable?

Reply:
Adjusted it like suggested in https://github.com/mtlynch/resticpy/pull/201#discussion_r2547536727